### PR TITLE
LPS-193230 [CP 2.0] [Bug] Fixed Project with unlimited seats count in team members cannot edit members from normal role to seats role

### PR DESCRIPTION
**Pull request by samuelleolopes**
[LPS-193230](https://liferay.atlassian.net/browse/LPS-193230)

### CODE REVIEW

**Comment by samuelleolopes** [comment]
ci:test:sf

**Comment by samuelleolopes** [comment]
ci:test:relevant

**Review by kylembischof — APPROVED** [review]
Looks good!

**Comment by samuelleolopes** [comment]
ci:test:sf

**Comment by samuelleolopes** [comment]
ci:test:relevant

**Review by EugenioAntunes — APPROVED** [review]
Nice job, @samuelleolopes  ⚡

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/TeamMembers/components/TeamMembersTable/TeamMembersTable.jsx [hunk 1]
```diff
@@ -83,6 +106,79 @@ const TeamMembersTable = ({
...
 	);
 
+	const mapFilterToContactsCategory = (filter) => {
+		const lowerCaseFirstLetter =
+			filter.charAt(0).toLowerCase() + filter.slice(1);
+
+		return {
+			contactsCategory: {
+				key: lowerCaseFirstLetter.replace(/\s/g, ''),
+				name: `${filter}`,
```
Comment (kevenleone) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/TeamMembers/components/TeamMembersTable/TeamMembersTable.jsx:116]:
```suggestion
				name: filter,
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/TeamMembers/components/TeamMembersTable/TeamMembersTable.jsx [hunk 2]
```diff
@@ -83,6 +106,79 @@ const TeamMembersTable = ({
...
+			try {
+				const updatedFilteredContacts = {};
+
+				for (const filter of Object.keys(
+					HIGH_PRIORITY_CONTACT_CATEGORIES
+				)) {
+					const contacts = await getHighPriorityContactsByFilter(
+						filter
+					);
+					updatedFilteredContacts[filter] = contacts;
```
Comment (kevenleone) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/TeamMembers/components/TeamMembersTable/TeamMembersTable.jsx:162]:
```suggestion
					updatedFilteredContacts[filter] = await getHighPriorityContactsByFilter(
						filter
					);
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/TeamMembers/components/TeamMembersTable/components/columns/OptionsColumn/OptionsColumn.jsx [hunk 3]
```diff
@@ -36,16 +45,26 @@ const OptionsColumn = ({
...
+						}
+						items={userOptions}
+						menuElementAttrs={{
+							className: 'p-0',
+						}}
+						menuWidth="shrink"
+					/>
+				</span>
+			</ClayTooltipProvider>
+		</>
```
Comment (kevenleone) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/TeamMembers/components/TeamMembersTable/components/columns/OptionsColumn/OptionsColumn.jsx:67]: If the fragment makes no difference here, remove plz

```suggestion
		<ClayTooltipProvider>
			<span>
				<ButtonDropDown
					customDropDownButton={
						<ButtonWithIcon
							displayType="null"
							small
							symbol="ellipsis-v"
						/>
					}
					items={userOptions}
					menuElementAttrs={{
						className: 'p-0',
					}}
					menuWidth="shrink"
				/>
			</span>
		</ClayTooltipProvider>
```

**Review by pedroliveirav — APPROVED** [review]

**Review by kevenleone — CHANGES_REQUESTED** [review]

**Comment by samuelleolopes** [comment]
ci:test:sf

**Comment by samuelleolopes** [comment]
ci:test:relevant

**Review by kevenleone — APPROVED** [review]
LGTM 🔥 

Good job @samuelleolopes , @Aline160

**Comment by kevenleone** [comment]
ci:forward

**Comment by kevenleone** [comment]
Just starting reviewing :)

**Pull request closed without merging**